skus/controllers.go

```diff
@@ -869,9 +869,58 @@ func VerifyCredentialV1(service *Service) handlers.AppHandler {
 func WebhookRouter(service *Service) chi.Router {
 	r := chi.NewRouter()
 	r.Method("POST", "/stripe", middleware.InstrumentHandler("HandleStripeWebhook", HandleStripeWebhook(service)))
+	r.Method("POST", "/android", middleware.InstrumentHandler("HandleAndroidWebhook", HandleAndroidWebhook(service)))
 	return r
 }
 
+// HandleAndroidWebhook is the handler for stripe checkout session webhooks
+func HandleAndroidWebhook(service *Service) handlers.AppHandler {
```

Comment:
Are there any tests for this. even just a happy path to test out the GetOrderByExternalID

Reply:
https://github.com/brave-intl/bat-go/pull/1558/commits/4ec182f93df86e8e11b4b3fcf50243301414e148 adds a webhook controller test
https://github.com/brave-intl/bat-go/pull/1558/commits/1f332ef8be68974360f74e91c43396a05bb03b52  adds a get by external id test